lib/bundler/cli/gem.rb

```diff
@@ -29,6 +29,11 @@ def run
       constant_name = name.gsub(/-[_-]*(?![_-]|$)/) { "::" }.gsub(/([_-]+|(::)|^)(.|$)/) { $2.to_s + $3.upcase }
       constant_array = constant_name.split("::")
 
+      if !Bundler.git_present?
+        Bundler.ui.error "You need to install git to be able to use bundle gem."
```

Comment:
Running `bundle gem` without git installed should work as expected for the user. We should just be setting empty values for `author`, `username` and `email` if we cannot source it from git.